src/M6Web/Bundle/LogBridgeBundle/DependencyInjection/Configuration.php

```diff
@@ -17,8 +17,8 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('m6web_log_bridge');
+        $treeBuilder = new TreeBuilder('m6web_log_bridge');
+        $rootNode = $treeBuilder->getRootNode();
```

Comment:
I think we should `require` at least `"symfony/config": "^4.2"` since this method does not exist before

https://github.com/symfony/config/blob/4.2/Definition/Builder/TreeBuilder.php#L57